app/helpers/accessible_link_helper.rb

```diff
@@ -4,4 +4,10 @@ def link_to_with_hidden_suffix(path:, text:, suffix:, klass: nil)
       "#{text}<span class='govuk-visually-hidden'> #{suffix}</span>".html_safe
     end
   end
+
+  def button_to_with_hidden_suffix(path:, text:, method:, params:, suffix:, klass: nil)
```

Comment:
@mpw5 don't need a test for this method do we?

Reply:
🤦 Excellent point. It has one now.

Feel free to re-review 👍